shuffle/cli/src/account.rs

```diff
@@ -13,17 +13,42 @@ use diem_sdk::{
 use diem_types::{
     account_config, chain_id::ChainId, transaction::authenticator::AuthenticationKey,
 };
+use dir::home_dir;
 use generate_key::load_key;
 use shuffle_transaction_builder::framework::encode_create_parent_vasp_account_script_function;
-use std::path::PathBuf;
+use std::path::{Path};
 
-pub fn handle(project_dir: PathBuf, account_key_path: PathBuf) -> Result<()> {
-    let config_path = project_dir.join("nodeconfig/0").join("node.yaml");
+
+pub fn handle(rotate : bool) -> Result<()> {
```

Comment:
I suppose if the user has already created an account under some address 0x42, and calls `account` command again with `rotate` flag, we should keep the same account under that address 0x42 and just rotate the authentication key stored in that account on-chain, right? However, right now `rotate` just creates a new account under a new address. What we can do is to send a `rotate_authentication_key` transaction if `rotate` flag is set.

Reply:
Yes you're right. I'll address this in my accounts pr #9296